include/rangers-api/Application/Player/PlayerStateBase.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+namespace app::player {
+    class PlayerStateBase : public GOCPlayerStateBase<PlayerHsmContext> {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.